ServerCore/Startup.cs

```diff
@@ -59,6 +59,12 @@ public void ConfigureServices(IServiceCollection services)
             {
                 options.Conventions.AuthorizeFolder("/Pages");
                 options.Conventions.AuthorizeFolder("/ModelBases");
+            }).AddViewOptions(options =>
+            {
+                if (Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT") == Environments.Development)
```

Comment:
Please add a comment here documenting exactly why this is in place :-)

Reply:
(to be clear, I fully support this approach over modifying jQuery!!! I just want us to remember why this is needed)